### PR TITLE
Forms Automated Test - LPS-133730 | master

### PR DESCRIPTION
**Pull request by ewertonluna**

### CODE REVIEW

**Comment by mlan0** [comment]
ci:test:sf

**Comment by mlan0** [comment]
Hi @ewertonluna , can you please rebase using the latest master to fix the conflicts? Thanks in advance.

**Comment by ewertonluna** [comment]
> Hi @ewertonluna , can you please rebase using the latest master to fix the conflicts? Thanks in advance.

done!

File: portal-web/test/functional/com/liferay/portalweb/paths/portlet/forms/Form.path [hunk 1]
```diff
@@ -21,10 +21,15 @@
 	<td></td>
 </tr>
 <tr>
-	<td>SAVE_NOTIFICATION</td>
-	<td>//div[contains(@class,'autosave-bar')]//span</td>
```
Comment (mlan0) [inline portal-web/test/functional/com/liferay/portalweb/paths/portlet/forms/Form.path:25]: Hi @ewertonluna , do you need to remove this code?

**Comment by mlan0** [comment]
ci:test:sf

File: portal-web/test/functional/com/liferay/portalweb/paths/portlet/forms/Form.path [hunk 2]
```diff
@@ -31,10 +31,15 @@
...
+	<td>FORM_TEMPORARILY_NOT_AVAILABLE</td>
+	<td>//div[contains(concat(' ',normalize-space(@class),' '),'alert')][contains(concat(' ',normalize-space(@class),' '),'alert-danger')][contains(text(),'Form is temporarily unavailable.')]</td>
 	<td></td>
 </tr>
+<tr>
+    <td>SAVE_NOTIFICATION</td>
+    <td>//div[contains(@class,'autosave-bar')]//span</td>
+    <td></td>
```
Comment (mlan0) [inline portal-web/test/functional/com/liferay/portalweb/paths/portlet/forms/Form.path:41]: Hi @ewertonluna, the indentation for the SAVE_NOTIFICATION is not the same as the others.

**Comment by mlan0** [comment]
ci:forward

**Pull request closed without merging**